src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -54,6 +80,7 @@ def handle_progression_build_not_found(
   clears progression_pending testcase metadata"""
   testcase_id = uworker_output.uworker_input.testcase_id
   testcase = data_handler.get_testcase_by_id(testcase_id)
+  _maybe_clear_progression_last_min_max_metadata(testcase, uworker_output)
```

Comment:
This is OK for now, but I guess we have to do this before error handling (ugh, bad assumptions).